open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/synchronization/beans/Referenceable.java

```diff
@@ -127,6 +158,8 @@ public String toString()
      *
      * @param objectToCompare supplied object
      * @return boolean result of comparison
+     * 
+     * GUID is not used in equals(...) to compare beans from repository and request. 
```

Comment:
Why is GUID not used - seems that the request should not have a GUID property since it is not to be set up by the caller

Reply:
yes, that is the case.